src/fetch/rewards.py

```diff
@@ -0,0 +1,167 @@
+"""Functionality for computing solver rewards
+
+The only function used in other parts of the code is `compute_solver_rewards` for generating
+aggregate reward information per solver."""
+
+from pandas import DataFrame, Series
+from src.fetch.prices import TokenConversion
+
+EPSILON_UPPER = 12_000_000_000_000_000
```

Comment:
Why do we use a different convention here compared to the constants below, and not just write

`0.012 * 10**18`

Reply:
I changed it to `12 * 10**15`. Unfortunately, `0.012 * 10**18` is something different, i.e. a float.